index.ts

```diff
@@ -88,15 +97,30 @@ Store.open().then((store) => {
       user =  await directory.getUserById(userID)
     }
 
-    //Fill cache
+    // Fill cache
     users[userID] = user;
     res.json(user);
   });
 
-  app.get("/todos", checkJwt, middleware.Authz(), server.list.bind(server));
-  app.post("/todos", checkJwt, middleware.Authz(), server.create.bind(server));
-  app.put("/todos/:id", checkJwt, middleware.Authz(), server.update.bind(server));
-  app.delete("/todos/:id", checkJwt, middleware.Authz(), server.delete.bind(server));
+  // using restMiddleware.Authz() will dispatch to a custom policy module based on REST convention
```

Comment:
Consider "restMiddleware.Authz() selects the policy module to evaluate based on REST convention"